app/src/main/res/layout/layout_keyboard.xml

```diff
@@ -86,7 +86,7 @@
             <TextView
                 android:id="@+id/btn_start"
                 style="@style/AppTheme.KeyboardButton"
-                android:layout_height="40dp"
+                android:layout_height="27dp"
```

Comment:
I think, There is no good idea decrease size of view, to make visible start word on specific device. Need other solution.